src/instruction.rs

```diff
@@ -94,3 +103,14 @@ pub fn noop() -> Instruction {
     data: PythClientInstruction::Noop.try_to_vec().unwrap(),
   }
 }
+
+// Returns ok if price is not stale
+pub fn price_status_check(price_account_data: Vec<u8>, expected_price_status: PriceStatus) -> Instruction {
```

Comment:
This could take a Price and serialize it to bytes internally, correct? that would be a nicer interface.

Reply:
Yes absolutely. Fixed.